cabal-install/cabal-install.cabal

```diff
@@ -90,6 +90,8 @@ Extra-Source-Files:
   tests/IntegrationTests2/targets/tests-disabled/q/q.cabal
   tests/IntegrationTests2/targets/variety/cabal.project
   tests/IntegrationTests2/targets/variety/p.cabal
+  tests/UnitTests/Distribution/Client/Init/goldens/generateCabalFile/exe-only.cabal
+  tests/UnitTests/Distribution/Client/Init/goldens/generateCabalFile/lib-exe-and-test.cabal
```

Comment:
they could be somewhere less burried. `tests/fixtures/init/*.cabal` is "deep" enough.

I'd like to avoid mixing "data" and "src" directories.

Reply:
Done.